samples/bot-conversation/nodejs/bots/teamsConversationBot.js

```diff
@@ -33,6 +34,8 @@ class TeamsConversationBot extends TeamsActivityHandler {
                 await this.messageAllMembersAsync(context);
             } else if (text.includes('who')) {
                 await this.getSingleMember(context);
+            } else if (text.includes('immersivereader')) {
```

Comment:
I believe we need to update the welcome message adaptive card to include this option. 
![image](https://user-images.githubusercontent.com/31851992/209806488-5eb16501-372b-4ab2-82eb-f14ddaac4c63.png)

Reply:
Added